XForm/XForm.Test/Commands/CosmosUnescapeTests.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Microsoft.CodeAnalysis.Elfie.Model.Strings;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Collections.Generic;
+using System.Diagnostics;
+using XForm.Commands;
+using XForm.Data;
+using XForm.Query;
+
+namespace XForm.Test.Commands
+{
+    [TestClass]
+    public class CosmosUnescapeTests
+    {
+        [TestMethod]
+        public void CosmosUnescapeCommand()
+        {
+            // The dictionary key is the input text, the value is the expected unescaped text.
+            Dictionary<String8, String8> testCases = new Dictionary<String8, String8>();
+            testCases.Add(String8.Empty, String8.Empty);
+            testCases.Add(String8.Convert("#NULL#", new byte[6]), String8.Empty);
```

Comment:
You can allocate a String8Block and use String8Block.GetCopy(string) for each of these. It's much nicer than figuring out the byte array size for each one.